packaging/debian/nvidia-container-toolkit-base.postinst

```diff
@@ -7,16 +7,20 @@ case "$1" in
         /usr/bin/nvidia-ctk --quiet config --config-file=/etc/nvidia-container-runtime/config.toml --in-place
 
         # Enable nvidia-cdi-refresh services on both install and upgrade
-        if command -v systemctl >/dev/null 2>&1 \
-           && systemctl --quiet is-system-running 2>/dev/null; then
-
-          systemctl daemon-reload || echo "Warning: Failed to reload systemd daemon" >&2
-          systemctl enable --now nvidia-cdi-refresh.path || echo "Warning: Failed to enable nvidia-cdi-refresh.path" >&2
-          systemctl enable --now nvidia-cdi-refresh.service || echo "Warning: Failed to enable nvidia-cdi-refresh.service" >&2
-          
-          # Trigger CDI spec regeneration immediately after install/upgrade
-          echo "Regenerating NVIDIA CDI specification..."
-          systemctl start nvidia-cdi-refresh.service || echo "Warning: Failed to trigger CDI refresh" >&2
+        # Support running, degraded, and maintenance systemd states
+        if command -v systemctl >/dev/null 2>&1; then
+          SYSTEMD_STATE=$(systemctl is-system-running 2>/dev/null || true)
+          case "$SYSTEMD_STATE" in
+            running|degraded|maintenance)
```

Comment:
Question: What does `maintenance` mean in this context. SHOULD we be generating specs in this state?

Reply:
From docs:

> System Maintenance Mode is a temporary status for a computer or software system where user access is restricted or blocked to allow administrators to perform essential updates, backups, or other maintenance tasks safely. During this mode, the system often displays a message informing users that it is unavailable and will be back online once the maintenance is complete.

So I think a user should not be installing things during a `maintenance` period